lib/mayaUsdAPI/CMakeLists.txt

```diff
@@ -52,6 +52,11 @@ if (UFE_CLIPBOARD_SUPPORT)
     list(APPEND HEADERS
         clipboard.h
     )
+
+    target_compile_definitions(${PROJECT_NAME}
+        PRIVATE
+            UFE_CLIPBOARD_SUPPORT=1
+    )
```

Comment:
Had some test crashes in my LookdevX PR when updating to this enhanced MayaUsdAPI because this define was missing.

Reply:
BTW, the test crashes only happened on Linux/Mac. On Windows all tests passed.